server/src/http/unleash_client.rs

```diff
@@ -285,15 +286,20 @@ impl UnleashClient {
     }
 
     fn client_features_req(&self, req: ClientFeaturesRequest) -> RequestBuilder {
-        let client_req = self
+        let mut client_req = self
```

Comment:
We don't need this, right?

```suggestion
        let client_req = self
```